### DISCUSSION TITLE
stepper motor

### DISCUSSION THREAD

**Discussion by Ruggyo** [nliaudat/floor-heating-proportional-valve#7] [Ideas]
Hi,

what do you think using stepper motor like this?
https://components101.com/motors/28byj-48-stepper-motor
in this case you can avoid the position sensor after a calibration
and probabily is cheaper

Bye
Ruggiero

**Comment by nliaudat** [comment]
Hi, 
The problem is that valve actuators need a lot of torque (approx 10kg).
Your stepper as only 300 gf.cm
